UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.UnitTests/Services/ProductDataServiceTests.cs

```diff
@@ -374,12 +447,69 @@ public async Task WhenValidProductVersionRequest_ThenValidateProductDataByProduc
             Assert.IsTrue(result.IsValid);
         }
 
+        [Test]
+        public async Task WhenValidProductVersionRequestFileSizeLargewithB2CToken_ThenCreateProductDataByProductVersionsReturnsBadRequest()
+        {
+            fakeAzureAdB2CConfig.Value.ClientId = "9bca10f0-20d9-4b38-99eb-c7aff6b5f551";
```

Comment:
Can we declare below in SetUp method instead of declaring in individual test cases.
fakeAzureAdB2CConfig.Value.ClientId = "9bca10f0-20d9-4b38-99eb-c7aff6b5f551";
            fakeAzureAdB2CConfig.Value.Instance = "https://gk.microsoft.com/";
            fakeAzureAdB2CConfig.Value.TenantId = "9b29766b-896f-46df-8f3a-122d7c822d01";
            fakeAzureAdConfig.Value.MicrosoftOnlineLoginUrl = "https://www.microsoft.com/";
            fakeEssFulfilmentStorageConfig.Value.LargeExchangeSetSizeInMB = 300;

Also change test case name WhenLargeExchangeSetRequestedWithB2CToken_ThenCreateProductDataByProductVersionsReturnsBadRequest
WhenLargeExchangeSetRequestedWithADB2CToken_ThenCreateProductDataByProductVersionsReturnsBadRequest